tests/neuro_san/zzz_hocons/test_smoke_test_hocons.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+from unittest import TestCase
+
+import pytest
+
+from parameterized import parameterized
+
+from neuro_san.test.unittest.dynamic_hocon_unit_tests import DynamicHoconUnitTests
+
+
+class TestSmokeTestHocons(TestCase):
```

Comment:
Move this class under zzz_hocons so that all the different kinds of tests live together and can be found and compared more easily.  We did not need yet another test_hocons directory.

Now, you might ask, why is this called "zzz_hocons"?
Answer: pytest tends to run tests in alphabetical order and I am looking for the regular unit tests to all run before the hocon tests, as they are more system-level tests.